packages/salesforcedx-vscode-lwc/test/vscode-integration/commands/forceLightningLwcStart.test.ts

```diff
@@ -279,7 +279,7 @@ describe('forceLightningLwcStart', () => {
         );
 
         sinon.assert.neverCalledWith(
-          devServiceStub.setBaseUrl,
+          devServiceStub.setBaseUrlFromDevServerUpMessage,
```

Comment:
This test was incorrect and should've been checking for `setBaseUrlFromDevServerUpMessage` method.